tests/integration/README.md

```diff
@@ -0,0 +1,181 @@
+# Integration tests
+
+## Prepare testing environment (tested on Fedora 33) 
+
+## Step 1 - Install docker and docker-compose for building RPMs locally
+
+The goal of this step is to have minimum the following versions of:
+```bash
+docker --version                                                                                                    
+# Docker version 20.10.6, build 370c289
+docker-compose --version                                                                                           
```

Comment:
@ZhukovGreen you should not use `docker-compose` otherwise your docker containers will not be compatible with `podman`.

Reply:
It should be: https://www.redhat.com/sysadmin/podman-docker-compose

Reply:
@bocekm  that's great! But it's just since podman 3, so it will be possible just on Fedora 33+. RHEL 8 is out of game.

Reply:
That's right. Yet, I'd call this limitation an ok aspect of the first iteration. We can make it compatible with other platforms in the subsequent iterations.